test_scripts/API/AutoCompleteList/002_ATF_SetGlobalProperties_P_AutoCompleteList_WithOutParam.lua

```diff
@@ -0,0 +1,82 @@
+--------------------------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL SDL respond <success = true, resultCode = "SUCCESS"> to mobile app
+--
+-- Description:
```

Comment:
@Aleksandriys,

    Description currently looks like "expected result". Please try to use description describing case: "Case when ..."
    please update description with particular resultCode from HMI (please specify "SUCCESS") and not <resultCode_received_from _HMI>